### PR TITLE
Release/v0.33.0

### PR DESCRIPTION
**Pull request by collindutter**
- [x] I have read and agree to the contributing guidelines for [submitting new pull requests](https://github.com/griptape-ai/griptape?tab=readme-ov-file#submitting-pull-requests).

## Describe your changes
Release `0.33.0`
## Issue ticket number and link
NA

----
📚 Documentation preview 📚: https://griptape--1243.org.readthedocs.build//1243/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,87 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- **BREAKING**: Renamed parameter `pipe` on `HuggingFacePipelinePromptDriver` to `pipeline`.
+- **BREAKING**: Update `pypdf` dependency to `^5.0.1`.
+- **BREAKING**: Update `redis` dependency to `^5.1.0`.
+- **BREAKING**: Removed `BaseFileManager.default_loader` and `BaseFileManager.loaders`.
+- **BREAKING**: Loaders no longer chunk data, use a Chunker to chunk the data.
+- **BREAKING**: Removed `fileutils.load_file` and `fileutils.load_files`.
+- **BREAKING**: Removed `loaders-dataframe` and `loaders-audio` extras as they are no longer needed.
+- **BREKING**: `TextLoader`, `PdfLoader`, `ImageLoader`, and `AudioLoader` now take a `str | PathLike` instead of `bytes`. Passing `bytes` is still supported but deprecated.
+- **BREAKING**: Removed `DataframeLoader`.
+- **BREAKING**: Update `pypdf` dependency to `^5.0.1`.
```
Comment (cjkindel) [inline CHANGELOG.md:59]: duplicate pypdf & redis entries (L51-52)

Comment (collindutter) [reply CHANGELOG.md:57]: Good catch, updated

File: MIGRATION.md [hunk 2]
```diff
@@ -1,8 +1,268 @@
...
+pip install griptape[drivers-prompt-huggingface-hub]
+```
+
+#### After
+```bash
+pip install griptape[drivers-prompt-huggingface-hub]
+pip install torch
+```
+
+### Removed `MediaArtifact`
```
Comment (vasinov) [inline MIGRATION.md:21]: Duplicate?

**Review by vasinov — APPROVED** [review]

File: MIGRATION.md (hunk 2 above)
Comment (collindutter) [reply MIGRATION.md]: Yes, migration guide was messed up.

**Review by vasinov — APPROVED** [review]

**Pull request merged**